### PR TITLE
123 settings my accounts page

### PR DESCRIPTION
**Pull request by chris-saburo**
Done it.

### CODE REVIEW

File: src/views/ArticleAdd.vue [hunk 1]
```diff
@@ -100,6 +100,7 @@ export default {
       const params = {
         body: {
           date: date_time,
+          categoryId: this.get_selected_category.id,
```
Comment (atharva3010) [inline src/views/ArticleAdd.vue:103]: Why have you added this?

File: src/components/BaseUserAccountInfo.vue [hunk 2]
```diff
@@ -6,10 +6,10 @@
       </h2>
     </div>
     <div class="mb-4">
-      <BaseInputText class="bg-surface" label="Name" type="text"></BaseInputText>
+      <BaseInputText class="bg-surface" label="Name" type="text" v-model="username"></BaseInputText>
     </div>
     <div class="mb-4">
-      <BaseInputText class="bg-surface" label="Email" type="email"></BaseInputText>
+      <BaseInputText class="bg-surface" label="Email" type="email" v-model="password"></BaseInputText>
```
Comment (atharva3010) [inline src/components/BaseUserAccountInfo.vue:12]: It is for email, not password

Comment (atharva3010) [reply src/components/BaseUserAccountInfo.vue:12]: We don't need a field for password

Comment (atharva3010) [reply src/components/BaseUserAccountInfo.vue:12]: We only need "Name" and "Email" fields

Comment (chris-saburo) [reply src/components/BaseUserAccountInfo.vue]: Done it.

File: src/views/ArticleAdd.vue (hunk 1 above)
Comment (atharva3010) [reply src/views/ArticleAdd.vue:103]: can you remove this?

Comment (chris-saburo) [reply src/views/ArticleAdd.vue:103]: it doesn't exist it here.

Comment (chris-saburo) [reply src/views/ArticleAdd.vue:103]: Done it.

**Comment by atharva3010** [comment]
You have to create `update_email` and `update_name` mutations to update values to the store. Then you have to create a computed property with `get()` (which returns the getter) and `set()` (which triggers the mutation). I hope this makes sense.

**Comment by chris-saburo** [comment]
> You have to create `update_email` and `update_name` mutations to update values to the store. Then you have to create a computed property with `get()` (which returns the getter) and `set()` (which triggers the mutation). I hope this makes sense.
> 
> Right now the code is `v-model` to the getters directly which does not make sense at all as getters do not trigger updates.

Done it.

File: src/components/BaseUserAccountInfo.vue [hunk 3]
```diff
@@ -20,15 +20,35 @@
...
 export default {
   name: 'BaseUserAccountInfo',
   components: { BaseInputText, BaseButton },
+  methods: {
+    ...mapMutations('auth', ['update_name', 'update_email'])
+  },
   computed: {
-    ...mapGetters('auth', ['email', 'password'])
+    ...mapGetters('auth', ['name', 'email']),
+    name: {
```
Comment (atharva3010) [inline src/components/BaseUserAccountInfo.vue:36]: Your computed properties have same name as getters, please use a different name

Comment (chris-saburo) [reply src/components/BaseUserAccountInfo.vue]: Done it.

File: src/components/BaseUserAccountInfo.vue [hunk 4]
```diff
@@ -20,15 +20,35 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex';
+import store from '@/store';
```
Comment (atharva3010) [inline src/components/BaseUserAccountInfo.vue:23]: Why have you imported the whole store?

Comment (chris-saburo) [reply src/components/BaseUserAccountInfo.vue:23]: Done it.

**Pull request merged**